### ISSUE TITLE
Add seed plumbing from experiment_summary.yaml to generated torchtune configs

### ISSUE THREAD

**Issue by niznik-dev** [niznik-dev/predicting-zygosity#417]
## Problem

`tools/torchtune/setup_finetune.py` has no way to set a training seed per run. The torchtune config template at `tools/torchtune/templates/finetune_template.yaml:131` hardcodes `seed: 1234`, and while the custom recipes correctly read `cfg.seed` and call `torchtune.training.set_seed()`, there's no plumbing from `experiment_summary.yaml` to override the seed on a per-run basis.

This means seed-variance experiments (e.g., "does finding X replicate across training seeds?") currently require hand-editing each generated `finetune.yaml` after scaffolding, which is error-prone and undocumented.

## Proposed fix

Allow `seed` as a field in `runs[].parameters` in experiment_summary.yaml. When present, setup_finetune.py should write it into the generated finetune.yaml in place of the template default, respecting the existing precedence ladder (CLI > config_file > recipe > argparse_default).

Example usage:
```yaml
runs:
  - name: Llama-3.2-3B-Instruct_r8_s7
    type: fine-tuned
    model: Llama-3.2-3B-Instruct
    parameters:
      seed: 7
  - name: Llama-3.2-3B-Instruct_r8_s23
    type: fine-tuned
    model: Llama-3.2-3B-Instruct
    parameters:
      seed: 23
```

## Scope

- Add `seed` to setup_finetune.py's argparse args
- Read seed from `runs[].parameters.seed` when present
- Write it into the generated torchtune yaml
- Short note to experiment_summary.yaml schema docs if they exist

## Motivation

Discovered while scoping a seed-stability check for the majority sanity-check experiment (blog post work related to #370). The check needs 5 training runs differing only in random seed. Current plan is to hand-edit the generated configs as a one-off workaround; a proper fix would make this a first-class cruijff_kit workflow.

**Comment by niznik-dev** [comment]
@sarahepedersen @msalganik This would be a good point to add any other parameters we've noticed ourselves changing in experiments that weren't obvious before
